plugins/cilium-cni/cni-install.sh

```diff
@@ -3,7 +3,15 @@
 set -eu
 
 HOST_PREFIX=${HOST_PREFIX:-/host}
-CNI_CONF_NAME=${CNI_CONF_NAME:-05-cilium.conf}
+if [ -z "${CILIUM_FLANNEL_MASTER_DEVICE}" ]; then
+	CNI_CONF_NAME=${CNI_CONF_NAME:-05-cilium.conf}
+else
+	until ip link show "${CILIUM_FLANNEL_MASTER_DEVICE}" &>/dev/null ; do
+		echo "Waiting for ${CILIUM_FLANNEL_MASTER_DEVICE} to be initialized"
+		sleep 1s
+	done
+	CNI_CONF_NAME=${CNI_CONF_NAME:-04-flannel-cilium-cni.conflist}
```

Comment:
Document in the corresponding file that this script (`cni-install.sh`) is dependent upon this name, e.g., in `04-flannel-cilium-cni.conflist` that `plugins/cilium-cni/cni-install.sh is dependent upon the name of this file being 04-flannel-cilium-cni.conflist`.